src/hotspot/share/jvmci/jvmciCodeInstaller.cpp

```diff
@@ -650,6 +651,53 @@ void CodeInstaller::initialize_dependencies(HotSpotCompiledCodeStream* stream, u
   }
 }
 
+JVMCI::CodeInstallResult CodeInstaller::install_runtime_stub(CodeBlob*& cb,
+                                                             const char* name,
+                                                             CodeBuffer* buffer,
+                                                             int stack_slots,
+                                                             JVMCI_TRAPS) {
+  if (name == nullptr) {
+    JVMCI_ERROR_OK("stub should have a name");
+  }
+
+  name = os::strdup(name); // Note: this leaks. See JDK-8289632
```

Comment:
JDK-8289632 was closed as not an issue since RuntimeStubs are never freed in normal execution so maybe drop this comment.